src/lens.c

```diff
@@ -1791,6 +1793,81 @@ void _prop_lv_lens_request_update()
      */
     prop_request_change(PROP_LV_LENS, &lv_lens_raw, 0);
 }
+#endif
+
+#ifdef CONFIG_DIGIC_VIII
+#define DYNAMIC_FLAG_ST2_MF    0x80  // true -> MF, false -> AF
+#define DYNAMIC_FLAG_ST3_IS    0x0F  // looks PROP_LV_LENS_STABILIZE equiv.
```

Comment:
Yes, `DYNAMIC_` prefix as it applies to `LENS_DYNAMC_DATA`. `ST2` and `ST3` in fact are field names in structure. 

`lens_info.IS               = (_dynamic->st3 & DYNAMIC_FLAG_ST3_IS);`

Uppercase IS is already used in ML owned `lens_info` structure. So while confusing, this + it being a define were the factors why I used that as a part of this name.

Proposed names make much more sense, however I'd still prefix them with something so it is clear those apply to `LENS_DYNAMIC_DATA`

Reply:
lens_info.IS is not too bad because there's no confusion with the rest of the name, it's the whole name.

Oh and can't use BIT, my brain was confused by 0x80 and saw 0xf as a single bit.

If you can think of a nice name where the prefix doesn't make it crazy long, go for it.

Reply:
I decided to drop defines them at all. They are overkill in this case, made sense during development when I had multiple those defined for some fields.

See 87e1f34cd9b5aeeb47394a9d497508c5cb577d4b for update.

Reply:
Only used in one place and fairly obvious patterns for masks, I'm okay with this.